src/api/forms/form-metadata-repository.js

```diff
@@ -35,10 +35,10 @@ export async function listForms() {
  * @param {string} formId - ID of the form
  * @returns {Promise<FormConfiguration>} - form configuration
  */
-const getFormMetadata = async (formId) => {
+export const getFormMetadata = async (formId) => {
   const formMetadataFilename = getFormMetadataFilename(formId)
   const value = await readFile(formMetadataFilename)
```

Comment:
Could we read the file with `utf8` encoding?

Then Node.js can automatically `.toString()` the return value for us

```suggestion
  const value = await readFile(formMetadataFilename, { encoding: 'utf8' })
```